browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -1060,60 +1057,66 @@ void BraveBrowserView::UpdateWebViewRoundedCorners() {
   const auto in_split_view_mode =
       !!browser_->GetFeatures().split_view_browser_data();
 
-  auto update_corner_radius =
-      [in_split_view_mode](views::WebView* contents, views::WebView* devtools,
-                           DevToolsDockedPlacement devtools_placement,
-                           gfx::RoundedCornersF corners) {
-        // In addition to giving the contents container rounded corners, we also
-        // need to round the corners of the native view holder that displays the
-        // web contents.
-
-        // Devtools lies underneath the contents webview. Round all four
-        // corners.
-        if (devtools && devtools->holder()) {
-          devtools->holder()->SetCornerRadii(corners);
-        }
-
-        if (!in_split_view_mode) {
-          // In order to make the contents web view and devtools appear to be
-          // contained within a single rounded-corner view, square the contents
-          // webview corners that are adjacent to devtools.
-          // TODO(sko) We need to override
-          // BrowserView::GetDevToolsDockedPlacement(). It depends on coordinate
-          // of it but in split view mode, the calculation is not correct.
-          switch (devtools_placement) {
-            case DevToolsDockedPlacement::kLeft:
-              corners.set_upper_left(0);
-              corners.set_lower_left(0);
-              break;
-            case DevToolsDockedPlacement::kRight:
-              corners.set_upper_right(0);
-              corners.set_lower_right(0);
-              break;
-            case DevToolsDockedPlacement::kBottom:
-              corners.set_lower_left(0);
-              corners.set_lower_right(0);
-              break;
-            case DevToolsDockedPlacement::kNone:
-              break;
-            case DevToolsDockedPlacement::kUnknown:
-              break;
-          }
-        }
-
-        if (contents && contents->holder()) {
-          // Upstream uses layer for its background.
-          CHECK(contents->layer());
-          contents->layer()->SetRoundedCornerRadius(corners);
-          contents->holder()->SetCornerRadii(corners);
-        }
-      };
-
-  if (contents_container_view_) {
-    update_corner_radius(contents_container_view_->GetContentsView(),
-                         devtools_web_view_, devtools_docked_placement(),
-                         corners);
-  }
+  // auto update_corner_radius =
```

Comment:
can we please use fixup commits for this kind of stuff so we can rebase before review?